WalletWasabi.Fluent/ViewModels/AddWallet/AddedWalletPageViewModel.cs

```diff
@@ -1,21 +1,22 @@
 using ReactiveUI;
+using System.Reactive.Disposables;
+using System.Threading.Tasks;
+using WalletWasabi.Fluent.Models;
 using WalletWasabi.Fluent.Models.Wallets;
 using WalletWasabi.Fluent.ViewModels.Navigation;
 using WalletWasabi.Wallets;
-using System.Reactive.Disposables;
-using WalletWasabi.Fluent.Models;
-using System.Threading.Tasks;
 
 namespace WalletWasabi.Fluent.ViewModels.AddWallet;
 
-[NavigationMetaData(Title = "Success")]
 public partial class AddedWalletPageViewModel : RoutableViewModel
 {
 	private readonly IWalletSettingsModel _walletSettings;
 	private IWalletModel? _wallet;
 
 	private AddedWalletPageViewModel(IWalletSettingsModel walletSettings, WalletCreationOptions options)
 	{
+		Title = Lang.Resources.AddedWalletPageViewModel_Title;
```

Comment:
This looks wrong to me. We should use the `NavigationMetaData` attribute as before but instad of using "Success", use "AddedWalletPageViewModel_Title"

Reply:
I based this implementation on the fork. I will see if I can revert and add back the `NavigationMetaData`.
One thing is for sure, I will not be able to do a lot of back and forth with this PR, so we will decide a solution and stick with it.

Reply:
What I can do is remove the concept of Non Localized view models and instead make all Title, Caption implicit and only be visible in the resx file